userdocs/docs/grpc/services/microvm/v1alpha1/proto.md

```diff
@@ -0,0 +1,245 @@
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| microvm | [flintlock.types.MicroVM](#flintlock.types.MicroVM) |  |  |
+
+
+
+
+
+
+<a name="microvm.services.api.v1alpha1.ListMessage"></a>
+
+### ListMessage
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| microvm | [flintlock.types.MicroVMSpec](#flintlock.types.MicroVMSpec) |  |  |
+
+
+
+
+
+
+<a name="microvm.services.api.v1alpha1.ListMicroVMsRequest"></a>
+
+### ListMicroVMsRequest
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| namespace | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="microvm.services.api.v1alpha1.ListMicroVMsResponse"></a>
+
+### ListMicroVMsResponse
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| microvm | [flintlock.types.MicroVM](#flintlock.types.MicroVM) | repeated |  |
+
+
+
+
+
+
+<a name="microvm.services.api.v1alpha1.UpdateMicroVMRequest"></a>
+
+### UpdateMicroVMRequest
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| microvm | [flintlock.types.MicroVMSpec](#flintlock.types.MicroVMSpec) |  |  |
+| update_mask | [google.protobuf.FieldMask](#google.protobuf.FieldMask) |  |  |
+
+
+
+
+
+
+<a name="microvm.services.api.v1alpha1.UpdateMicroVMResponse"></a>
+
+### UpdateMicroVMResponse
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| microvm | [flintlock.types.MicroVMSpec](#flintlock.types.MicroVMSpec) |  |  |
+
+
+
+
+
+ 
+
+ 
+
+ 
+
+
+<a name="microvm.services.api.v1alpha1.MicroVM"></a>
+
+### MicroVM
+MicroVM providers a service to create and manage the lifecycle of microvms.
+
+| Method Name | Request Type | Response Type | Description |
+| ----------- | ------------ | ------------- | ------------|
+| CreateMicroVM | [CreateMicroVMRequest](#microvm.services.api.v1alpha1.CreateMicroVMRequest) | [CreateMicroVMResponse](#microvm.services.api.v1alpha1.CreateMicroVMResponse) |  |
+| UpdateMicroVM | [UpdateMicroVMRequest](#microvm.services.api.v1alpha1.UpdateMicroVMRequest) | [UpdateMicroVMResponse](#microvm.services.api.v1alpha1.UpdateMicroVMResponse) |  |
+| DeleteMicroVM | [DeleteMicroVMRequest](#microvm.services.api.v1alpha1.DeleteMicroVMRequest) | [.google.protobuf.Empty](#google.protobuf.Empty) |  |
+| GetMicroVM | [GetMicroVMRequest](#microvm.services.api.v1alpha1.GetMicroVMRequest) | [GetMicroVMResponse](#microvm.services.api.v1alpha1.GetMicroVMResponse) |  |
+| ListMicroVMs | [ListMicroVMsRequest](#microvm.services.api.v1alpha1.ListMicroVMsRequest) | [ListMicroVMsResponse](#microvm.services.api.v1alpha1.ListMicroVMsResponse) |  |
+| ListMicroVMsStream | [ListMicroVMsRequest](#microvm.services.api.v1alpha1.ListMicroVMsRequest) | [ListMessage](#microvm.services.api.v1alpha1.ListMessage) stream |  |
+
+ 
+
+
+
+## Scalar Value Types
+
+| .proto Type | Notes | C++ | Java | Python | Go | C# | PHP | Ruby |
```

Comment:
this table does not render all too nicely, the 2nd from left column is very squished so you get 1 word per line

Reply:
Auto-generated with `protobuf`

Reply:
hmmmm 😕